src/assets/custom/css/_sass/_website-header.scss

```diff
@@ -52,8 +54,9 @@ body:not(.toggle--slt-397-website-header-updates) .website-header {
         right: $lateral-space--medium;
       }
       @include large-and-extra-large {
-        left: $lateral-space--large;
-        right: $lateral-space--large;
+        display: block;
+        margin-left: $lateral-space--large;
```

Comment:
How would we look on super large screens? (Above 1800px)

Reply:
Good call. All good now: https://github.com/codurance/site/pull/1921/commits/7ebd3d2c1c7aee6b0ea7abab3676708cd840a8a2